frontend/src/views/RecoverPassword/RecoverPassword.js

```diff
@@ -0,0 +1,29 @@
+import {
+  Container,
+  Logo,
+  RecoverPasswordForm,
+  LogoSpan,
+  LogoContainer,
+  FormText,
+  EmailPlaceholder,
+  SubmitButton,
+} from './RecoverPassword.styled';
+
+const RecoverPassword = () => {
+  return (
+    <Container>
+      <LogoContainer>
+        <Logo to="/log-in">
+          Board<LogoSpan>MAP</LogoSpan>
+        </Logo>
+      </LogoContainer>
+      <RecoverPasswordForm>
+        <FormText>Enter Your E-mail to recover password!</FormText>
+        <EmailPlaceholder placeholder="Enter your email" />
```

Comment:
Jeżeli zamienisz te h2 na label, to można tu wtedy zrobić tak, żeby label działał. 
```suggestion
        <FormText>Enter Your E-mail to recover password!
            <EmailPlaceholder placeholder="Enter your email" />
        </FormText>
```